src/main/java/uk/gov/hmcts/reform/sscs/constants/WarningMessage.java

```diff
@@ -34,6 +34,9 @@
     HEARING_OPTIONS_EXCLUDE_DATES(SscsConstants.HEARING_OPTIONS_EXCLUDE_DATES, "Hearing options exclude dates"),
     APPELLANT_NINO(NINO, "Appellant nino"),
     APPELLANT_PHONE(PHONE, "Appellant phone"),
```

Comment:
Think you can probably remove this warning message now? (for appellant phone)

Reply:
Yes, removed from warning message